lib/ansible/modules/rpm_key.py

```diff
@@ -148,6 +152,15 @@ def __init__(self, module):
             else:
                 module.exit_json(changed=False)
 
+    def keyfile(self, key):
+        tmpfd, tmpname = tempfile.mkstemp()
```

Comment:
the module already has a private tmpdir, use that to keep files in a place we guarantee cleanup